### ISSUE TITLE
Bindings for touch_actions 

### ISSUE THREAD

**Issue by akinhwan** [pytest-dev/pytest-selenium#224]
Hello is there a way to utilize common touch_actions? 

https://seleniumhq.github.io/selenium/docs/api/py/webdriver/selenium.webdriver.common.touch_actions.html#selenium.webdriver.common.touch_actions.TouchActions.tap_and_hold

**Comment by BeyondEvil** [comment]
Yes. All the commands and actions that the selenium python bindings provide are available.
